pyresample/kd_tree.py

```diff
@@ -711,7 +726,7 @@ def get_sample_from_neighbour_info(resample_type, output_shape, data,
         if is_multi_channel:
             weight_funcs = weight_funcs * 2
         else:
-            weight_funcs = (weight_funcs,) * 2
+            weight_funcs = (weight_funcs, ) * 2
```

Comment:
What linter prefers this? Destroy it.

Reply:
it also says `r**2 / b**2` is better than `r ** 2 / b ** 2`, which I kind of like

Reply:
It's actually no a linter, it's yapf